uni-stark/src/prover.rs

```diff
@@ -147,6 +155,9 @@ where
         sels.inv_zeroifier.push(Val::<SC>::default());
     }
 
+    let mut alpha_powers = alpha.powers().take(constraint_count).collect_vec();
+    alpha_powers.reverse();
+
```

Comment:
Do we need to reverse here? It's a slightly different polynomial but it should be equally valid to do this without reversing.

Reply:
Hm I think the prover cost shouldn't be significant (since it's done once). It is a little less clean here to reverse, but I think we may want the verifier to stick with Horner's method so this order might be most natural there?

(Also just slightly hesitant to change it also affects Succinct's verifier program - probably not a big deal but just want to make sure it's a firm-ish decision.)